CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [0.5.2] (Unreleased)
 
 - Refactor `TableChain` to include `_searched` attribute. #867
+- Fix errors in config import #881
```

Comment:
I've been including PR numbers in changelog, instead of issue numbers. We could go either way, but I figure someone who wants to know about the details of a change would want to see the diff over the inciting issue
```suggestion
- Fix errors in config import #882
```